app/assets/stylesheets/modules/_results-section.scss

```diff
@@ -42,3 +42,30 @@
   margin: 0;
 }
 
+.paging {
+  clear: both;
+  padding: 20px;
+  text-align: center;
+  width: 100%;
+  margin: 0 auto;
+
+  a {
+    display: block;
+    padding: 14px 0;
+    background-color: #0182c5;
+    color: #fff;
+    height: 65px;
+    width: 65px;
+    text-align: center;
+    border-radius: 50%;
+    margin: 0 auto;
+
+    &:hover {
+      background-color: darken(#0182c5, 5%);
```

Comment:
Color literals like `#0182c5` should only be used in variable declarations; they should be referred to via variable everywhere else.